docs/doxygen/images/defender_demo.pu

```diff
@@ -0,0 +1,83 @@
+@startuml
+autonumber
+
```

Comment:
The MQTT and Shadow demo diagrams mention the library API functions used for different operations in the demos for the purpose of making the API functions utility clear for the reader. 
It would be good to at least mention the Defender library functions at their relevant use within the diagram

Reply:
Done.